bitnami/grafana-operator/Chart.yaml

```diff
@@ -29,5 +29,5 @@ maintainers:
   url: https://github.com/bitnami/charts
 name: grafana-operator
 sources:
-- https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
-version: 3.3.2
+  - https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
+version: 3.3.0
```

Comment:
@juan131
Hey,
it seems you actually _decreased_ the version number of bitnami/grafana-operator from 3.3.2 to 3.3.0. Did you mean 3.4.0?

Reply:
Good catch @BernhardGruen !! Sorry for this, I created so many PRs like this one that my brain wasn't very refreshed.
Fixing it at https://github.com/bitnami/charts/pull/18849